src/ring_theory/ideal/prod.lean

```diff
@@ -0,0 +1,189 @@
+/-
+Copyright (c) 2020 Markus Himmel. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Markus Himmel
+-/
+import ring_theory.ideal.operations
+import algebraic_geometry.prime_spectrum
```

Comment:
I'm not sure if we should have imports in this direction. Would you mind stating the lemmas on prime ideals in terms of `ideal.is_prime` and moving the stuff on prime spectrum to `algebraic_geometry.prime_spectrum`?

Reply:
I think I'll just remove all mentions of `prime_spectrum` altogether, since I've only used it as shorthand for `{ I : ideal R // I.is_prime }`.

Reply:
Sure, but someone will certainly want to use those facts in the language of `prime_spectrum`. So if you want to add this as "corollary" in `algebraic_geometry/` that would be good, I think.